cypress/integration/page.spec.js

```diff
@@ -64,509 +64,509 @@
-    });
+  //   it('should have navigation', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav().should('exist');
+  //   });
+
+  //   it('should mark Home nav link as active', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav()
+  //       .contains('a', 'Home')
+  //       .should('have.attr', 'href', '#/')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should not mark People nav link as active', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .should('have.attr', 'href', '#/people')
+  //       .and('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.assetTitle('Home Page');
+  //   });
+
+
+  //   it('should not send API request', () => {
+  //     page.spyOnPeopleRequest();
+  //     page.visit('/');
+  //     cy.get('@peopleRequest').should('not.be.called');
+  //   });
+
+  //   it('should not have loader', () => {
+  //     cy.clock();
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.loader().should('not.exist');
+  //   });
+
+  //   it('should not have the `no people` message', () => {
+  //     page.mockNoPeople();
+  //     page.visit('/');
+  //     page.noPeopleMessage().should('not.exist');
+  //   });
+
+  //   it('should not have loading error', () => {
+  //     page.mockPeopleError();
+  //     page.visit('/');
+  //     page.peopleLoadingError().should('not.exist');
+  //   });
+
+  //   it('should not have people table', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.peopleTable().should('not.exist');
+  //   });
+  // });
+
+  // describe('App', () => {
+  //   it('should redirect from /home to /', () => {
+  //     page.visit('/#/home');
+  //     page.assertHash('#/');
+  //     page.assetTitle('Home Page');
+  //   });
+  // });
+
+  // describe('Non existing page', () => {
+  //   it('should have correct address', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.assertHash('#/some/not/existing/page');
+  //   });
+
+  //   it('should have navigation', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.nav().should('exist');
+  //   });
+
+  //   it('should not have active nav links', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+
+  //     page.nav().contains('a', 'Home').should('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //     page.nav().contains('a', 'People').should('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.mockPeople();
+  //     page.visit('/#/some/not/existing/page');
+  //     page.assetTitle('Page not found');
```

Comment:
The function `assetTitle` is likely a typo and should be `assertTitle` to correctly describe its purpose of asserting the page title.